hub-templates/basehub/Chart.yaml

```diff
@@ -2,10 +2,9 @@ apiVersion: v2
 appVersion: '1.0'
 description: Deployment Chart for JupyterHub
 name: basehub
-# Let's keep this constant so other charts in this repo can depend on this easily
-version: 0.0.1-n569.hb296398
+version: "0.1.0"
 dependencies:
   - name: jupyterhub
-   # REMEMBER TO CHANGE BASE IMAGE OF images/hub/ WHEN CHANGING THIS
-    version: 0.11.1-n277.h61b0e003
+    # REMEMBER TO CHANGE BASE IMAGE OF images/hub/ WHEN CHANGING THIS
+    version: 1.0.0-beta.1
```

Comment:
Aha, amazing! I'm hoping we can extract this out to its own PR and merge it independently